stopcovid/clients/sqs.py

```diff
@@ -1,11 +1,17 @@
 import boto3
 import os
 import json
-from typing import List
 
+from typing import List
+from collections import defaultdict
 from stopcovid.event_distributor.outbound_sms import OutboundSMS
 
 
+def _get_message_deduplication_id(messages):
+    unique_message_ids = sorted(list(set([str(message.event_id) for message in messages])))
+    return "-".join(unique_message_ids)
```

Comment:
this works, though we'll soon have a batch ID that we can use

Reply:
ignore. You're potentially aggregating across batches.

Reply:
Do you think that the outbound sms flow should know about event batches at all? We flatten them out into events slightly upstream. I'll stew on it

Reply:
probably not... Agree with your instinct.